crates/oci-runtime-clap/README.md

```diff
@@ -0,0 +1,27 @@
+# oci-runtime-clap
+
+This is a crate to parse command line arguments for OCI container
+runtimes as specified in the [OCI Runtime Command Line
+Interface][https://github.com/opencontainers/runtime-tools/blob/master/docs/command-line-interface.md).
+
+## Implemented subcommands
+
+| Command    | oci-runtime-clap | CLI Specification | runc | crun | youki |
+| :--------: | :--------------: | :---------------: | :--: | :--: | :---: |
+| create     | ✅               | ✅                | ✅   | ✅   | ✅    |
+| start      | ✅               | ✅                | ✅   | ✅   | ✅    |
+| state      | ✅               | ✅                | ✅   | ✅   | ✅    |
+| kill       | ✅               | ✅                | ✅   | ✅   | ✅    |
+| delete     | ✅               | ✅                | ✅   | ✅   | ✅    |
+| :--------: | :--------------: | :---------------: | :--: | :--: | :---: |
```

Comment:
If you want to separate Common and Standard, why don't you write that down and separate them in a table?

Reply:
Sorry, I'm not sure what you mean.

Reply:
This doesn't seem to separate the tables. Is this the convention?
https://github.com/containers/youki/blob/deec552874e3d9b2ce2d2b3cb124853550cab347/crates/liboci-cli/README.md

Reply:
Oh, right, I see what you mean.  Clearly the `:---:` line doesn't mean what I thought it did in github markdown.  Should be fixed in the latest revision.